crates/mem-pool/src/deposit.rs

```diff
@@ -144,6 +146,20 @@ fn check_deposit_cell(ctx: &RollupContext, cell: &DepositInfo) -> Result<()> {
                 "Invalid deposit account script: unexpected hash_type: Data"
             ));
         }
+        // godwoken only allow to deposit to an EOA account
+        // check code hash of deposit reqeust is an EOA type hash
+        if !ctx
+            .rollup_config
+            .allowed_eoa_type_hashes()
+            .into_iter()
+            .any(|type_hash| type_hash.hash() == script.code_hash())
+        {
+            return Err(anyhow!(
+                "Not an EOA type hash: {}, we can only deposit to EOA account",
```

Comment:
```suggestion
                "Invalid deposit account script: script.code_hash is not in configured allowed_eoa_type_hashes, script.code_hash: {}"
```

Just follow the same style as other error logs, with "Invalid deposit account script: " prefix.